packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -382,6 +382,10 @@ class ScrollableBaseNative extends Component {
 			this.updateScrollbars();
 		}
 
+		if (isVerticalScrollbarVisible && !prevState.isVerticalScrollbarVisible) {
+			this.forceUpdate();
+		}
+
```

Comment:
Do we need this code in ui/Scrollable/Scrollable.js?

Reply:
Whoops forgot to delete this.